services/web-app/components/code/code.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { CodeSnippet, Column, Grid, Theme } from '@carbon/react'
+
+import styles from './code.module.scss'
+
+const Code = ({ children }) => (
+  <Theme theme={'g100'}>
+    <Grid>
+      <Column sm={4} md={6} lg={8}>
+        <CodeSnippet type="multi" wrapText="true" feedback="Copied!" className={styles.code}>
```

Comment:
React props passed with just their name have a value of true
```suggestion
        <CodeSnippet type="multi" wrapText feedback="Copied!" className={styles.code}>
```

<img width="471" alt="image" src="https://user-images.githubusercontent.com/40550942/165113807-d4eb20b7-816d-4153-aad7-fc800c81fb69.png">